modules/app.xql

```diff
@@ -59,6 +61,12 @@ declare function app:scheduler-enabled($node as node(), $model as map(*)) {
         $node
 };
 
+declare 
+    %templates:warp
+function app:java-version ($version as xs:string?, $model as map(*)) as node()? {
```

Comment:
because I don't know how to get the templating to work without an input parameter, but i m happy to switch to ``$node as node(), $model as map(*)`` if that would be better.